### PR TITLE
macOS conda instructions

### PR DESCRIPTION
**Pull request by alishakodibagkar**
Added a file called `iOS-install-and-tutorial-instructions-conda.md` under `brainlit/docs` to be linked in `README.md`

### CODE REVIEW

File: docs/iOS-install-and-tutorial-instructions-conda.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Installing brainlit on iOS using conda environment:
```
Comment (bvarjavand) [inline docs/iOS-install-and-tutorial-instructions-conda.md]: Mac, not iOS

File: docs/iOS-install-and-tutorial-instructions-conda.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+* `cd brainlit/docs/notebooks/utils`
+* run jupyter notebook: `jupyter notebook`
+* select `downloading_brains.ipynb`
+* run tutorial
+
+# Common issues and fixes:
+* Issues with using a jupyter notebook
+    * [fixes](https://jupyter-notebook.readthedocs.io/en/stable/troubleshooting.html)
+* Imports cause schema warning
+    * ignore this
```
Comment (bvarjavand) [inline docs/iOS-install-and-tutorial-instructions-conda.md]: This is too brief.

File: docs/iOS-install-and-tutorial-instructions-conda.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+* Activation key issue
+    * `cd ~/.cloudvolume/secrets`
+    * Add a .txt file containing aws access key and secret access key (more detailed instructions under README.md)
```
Comment (bvarjavand) [inline docs/iOS-install-and-tutorial-instructions-conda.md]: You can link to the markdown section (like in the Toc)

File: docs/iOS-install-and-tutorial-instructions-conda.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    * Convert .txt file to .json 
+* Section (1) Defining variables: Change directory names if they do not exist
```
Comment (bvarjavand) [inline docs/iOS-install-and-tutorial-instructions-conda.md]: This is outdated. Remove.

**Review by bvarjavand — CHANGES_REQUESTED** [review]
Good! A few changes but nothing major.

File: docs/iOS-install-and-tutorial-instructions-conda.md (hunk 1 above)
Comment (alishakodibagkar) [reply docs/iOS-install-and-tutorial-instructions-conda.md]: Changes "iOS" to "macOS" in name of file and headings

File: docs/iOS-install-and-tutorial-instructions-conda.md (hunk 2 above)
Comment (alishakodibagkar) [reply docs/iOS-install-and-tutorial-instructions-conda.md]: Changed "ignore this" to "This warning doesn't cause any errors, and the code can still run unaffected."

File: docs/macOS-install-and-tutorial-instructions-for-conda.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+2. Install brainlit
+    * Option 1: install from pypi
+        * install brainlit: `pip install brainlit`
+    * Option 2: install brainlit from source
+        * cd to directory you want to install brainlit in
+        * clone the repo: git clone https://github.com/neurodata/brainlit.git
+        * cd into the repo: `cd brainlit`
+        * install brainlit: `pip install -e .`
+
+# Downloading brains tutorial on iOS using conda environment, using jupyter notebook:
```
Comment (tathey1) [inline docs/macOS-install-and-tutorial-instructions-for-conda.md]: macOS not iOS

File: docs/iOS-install-and-tutorial-instructions-conda.md (hunk 3 above)
Comment (alishakodibagkar) [reply docs/iOS-install-and-tutorial-instructions-conda.md]: Linked to the document on AWS Credentials under `brainlit/docs`

File: docs/iOS-install-and-tutorial-instructions-conda.md (hunk 4 above)
Comment (alishakodibagkar) [reply docs/iOS-install-and-tutorial-instructions-conda.md]: Removed Section 1

File: docs/macOS-install-and-tutorial-instructions-for-conda.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+# Downloading brains tutorial on iOS using conda environment, using jupyter notebook: 
+* in a new terminal: `conda activate brainlit`
+* cd to directory brainlit is installed
+* `cd brainlit/docs/notebooks/utils`
+* run jupyter notebook: `jupyter notebook`
+* select `downloading_brains.ipynb`
+* run tutorial
+
+# Common issues and fixes:
```
Comment (tathey1) [inline docs/macOS-install-and-tutorial-instructions-for-conda.md]: Seems like this is "Common issues and fixes with downloading_brains notebook?" right? If so, please specify. If not, I'm not sure what you mean by "Section (2)" below

**Comment by bvarjavand** [comment]
Please rename `iOS-install-and-tutorial-instructions-conda.md` to `Mac-install-and-tutorial-instructions-conda.md` and update the link.

File: docs/macOS-install-and-tutorial-instructions-for-conda.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+* Issues with using a jupyter notebook
+    * [fixes](https://jupyter-notebook.readthedocs.io/en/stable/troubleshooting.html)
+* Imports section: cause schema warning
+    * This warning doesn't cause any errors, and the code can still run unaffected.
+* AWS Credendials Issues
+    * Refer to the instructions on: https://github.com/NeuroDataDesign/brainlit/blob/develop/docs/AWS_Credentials_Issues.md
+* Section (2) Create a Neuroglancer instance and download the volume: make sure variables are correct and functions have correct inputs
+    * Example 3: 
+        * Before: `img, bbox, vox = ngl_sess.pull_voxel(2, v_id, radius, radius, radius)`
+        * After:  `img, bbox, vox = ngl_sess.pull_voxel(2, v_id, radius)`
```
Comment (tathey1) [inline docs/macOS-install-and-tutorial-instructions-for-conda.md]: what does before and after mean? do you mean right and wrong?

**Review by tathey1 — CHANGES_REQUESTED** [review]
Please change the name of this PR to say macOS, not iOS

File: docs/macOS-install-and-tutorial-instructions-for-conda.md (hunk 5 above)
Comment (alishakodibagkar) [reply docs/macOS-install-and-tutorial-instructions-for-conda.md]: Changed PR name to macOS, and also renamed to `macOS-install-and-tutorial-instructions-conda.md`/changed iOS to macOS in all sections in newest commit

File: docs/macOS-install-and-tutorial-instructions-for-conda.md (hunk 7 above)
Comment (alishakodibagkar) [reply docs/macOS-install-and-tutorial-instructions-for-conda.md]: Changed "before" and "after" to "wrong" and "right"

File: docs/macOS-install-and-tutorial-instructions-for-conda.md (hunk 6 above)
Comment (alishakodibagkar) [reply docs/macOS-install-and-tutorial-instructions-for-conda.md]: Specified section title as "Common issues and fixes with downloading_brains notebook" and changed bullet point to "Section (2) of downloading_brains notebook" and "Section (4) of downloading_brains notebook"

**Review by tathey1 — CHANGES_REQUESTED** [review]
Content looks good, but could you please combine with https://github.com/NeuroDataDesign/brainlit/blob/develop/docs/macOS_Install_%26_Run_Issues.md ?

i.e. make a single md file that is for macOS users and combine your stuff with John's stuff

**Comment by alishakodibagkar** [comment]
Combined content with https://github.com/NeuroDataDesign/brainlit/blob/develop/docs/macOS_Install_%26_Run_Issues.md
@tathey1 @bvarjavand

**Review by tathey1 — APPROVED** [review]

**Review by bvarjavand — APPROVED** [review]

**Pull request merged**